src/asset-swapper/types.ts

```diff
@@ -196,7 +196,8 @@ enum ExchangeProxyRefundReceiver {
 export interface ExchangeProxyContractOpts {
     isFromETH: boolean;
     isToETH: boolean;
-    affiliateFees: readonly AffiliateFeeAmount[];
+    sellTokenAffiliateFees: readonly AffiliateFeeAmount[];
```

Comment:
This should be either (1) required and code that initialize `ExchangeProxyOpts` would need to pass `[]` for this argument or (2) optional